### DISCUSSION TITLE
Práctica Pasteleria (HTML) terminada

### DISCUSSION THREAD

**Discussion by PatricioNovellinoLaunchx** [Launch-X-Latam/MisionFrontEnd#298] [Show and tell]
Hola que tal a todos, pude terminar la práctica de pasteleria solamente con HTML y muy poco CSS, la voy a ir mejorando de a poco y además, es un proyecto personal de una conocida mía, asi que agradeceria cualquier feedback.

- Cliente: https://patricionovellino.github.io/Practica-2-HTML/pasteleria-cliente-practica2.html

- Admin: https://patricionovellino.github.io/Practica-2-HTML/pasteleria-admin-practica2.html

El repositorio de github esta acá: https://github.com/PatricioNovellino/PatricioNovellino.github.io/tree/main/Practica-2-HTML

**Comment by AbnerHub** [comment]
Amigo ¿se tiene que hacer una por del cliente y otra del Admin ?
 Las otras que he visto solo tienen una pagina

**Reply by PatricioNovellinoLaunchx** [reply]
Creo que se puede hacer de las dos formas

**Comment by RIvanCF** [comment]
Cuando tengas tiempo, sugiero que le metas CSS a tu pagina. 
Excelente diseño

**Comment by JhonRendonOsorio** [comment]
Disculpa. Cómo se sube a una web ?

**Reply by Standard-IO** [reply]
Hay muchas maneras, pero la verdad no te recomiendo que te metas a hacerlo ahora solo te vas a confundir. Probablemente , lo lleguemos a ver más adelante en el curso. 
Ahora bien, si no puedes esperar, el compañero utilizó el servicio de [Github Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages)  un servicio gratuito de github para hospedar páginas estáticas. Tiene sus limitaciones pero cumple con lo que estamos viendo ahora.

**Reply by JhonRendonOsorio** [reply]
Muchas gracias por la aclaración, ya estaba preocupado

**Comment by brayandoria98** [comment]
Me pareció muy buena. Igual para la práctica de HTML es opcional el CSS. Yo la hice sin CSS pero espero actualizarla en un futuro cuando sepa manejar bien el CSS. También hice dos páginas como tú porque eso entendí del enunciado.

**Comment by RIvanCF** [comment]
![Screenshot_2022-11-17-16-02-04-414_com.android.chrome.jpg](https://user-images.githubusercontent.com/99302791/202569416-0f9b25c6-308e-4392-9057-958eb622a7de.jpg)

Hay un error cuando la página se visualiza en un móvil. 
Te sugiero una leída a los responsive, versiones móbile y css

**Reply by PatricioNovellinoLaunchx** [reply]
Hola que tal, gracias por el comentario! El tema es que esa recien era la parte de html asi que no meti nada ni responsive. Ahora en poquito tiempo voy a subir la de css arreglando ese problema
